app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -393,6 +396,20 @@ private fun AssociationMembers(
                   user.value?.lastName?.let {
                     val lastName = it
                     Text("${firstName} ${lastName}")
+
+                    // Role Badge
+                    Box(
+                        modifier =
+                            Modifier.background(
+                                    color = Color(member.role.color),
+                                    shape = RoundedCornerShape(8.dp))
+                                .padding(horizontal = 8.dp, vertical = 4.dp)) {
+                          Text(
+                              text = member.role.displayName,
+                              style = MaterialTheme.typography.bodySmall,
+                              color = Color.White,
```

Comment:
You should make sure that the text color changes relatively to the color of the badge (For example, we couldn't see the text if badgeColorWhite was set for a role)

Reply:
Wow didn't think about that

Reply:
done !